iOS/Projects/Features/SignUp/Sources/View/NickNameBoxView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  NickNameBoxView.swift
+//  SignUpFeature
+//
+//  Created by 안종표 on 12/4/23.
+//  Copyright © 2023 kr.codesquad.boostcamp8. All rights reserved.
+//
+
+import Combine
+import CombineCocoa
+import DesignSystem
+import UIKit
+
+// MARK: - NickNameBoxView
+
+final class NickNameBoxView: UIView {
+  private var subscriptions: Set<AnyCancellable> = []
+
+  private let nickNameDidChangedSubject = PassthroughSubject<String, Never>()
+
+  var nickNameDidChangedPublisher: AnyPublisher<String, Never> {
+    return nickNameDidChangedSubject.eraseToAnyPublisher()
+  }
+
+  private let textField: UITextField = {
+    let textField = UITextField()
+    textField.translatesAutoresizingMaskIntoConstraints = false
+    textField.font = .systemFont(ofSize: 16, weight: .semibold)
+    textField.textColor = .black
```

Comment:
p2: 아래로 바꿔주세요! 안그러면 다크모드일 때 텍스트가 항상 검정으로 보입니다.
```suggestion
    textField.textColor = DesignSystemColor.primaryText
```

Reply:
피그마만보고 빨리빨리 진행하다보니 디테일한 부분들을 놓친것 같습니다 😓

Reply:
피그마만보고 빨리빨리 진행하다보니 피그마에 적용안돼있는 디테일한 부분들을 놓치게 된 것 같습니다 😓
바로 적용하겠습니다!